lib/beacon/live_admin/visual_editor/utils.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Beacon.LiveAdmin.VisualEditor.Utils do
+  def parse_number_and_unit(string) do
```

Comment:
Maybe move this function to `Beacon.LiveAdmin.VisualEditor` instead of creating a new module?